tests/note-editor.spec.ts

```
@@ -0,0 +1,109 @@
+  const notes = json?.result?.result ?? []
+
+  // For each note, check it's in the local Dexie DB
+  const exists = await page.evaluate(async (notes) => {
+    for (const note of notes) {
+      const dexieDB = createDexieDB()
+      const found = await dexieDB.notes.get(note.id)
+
+      return !!found
+    }
+  }, notes)
+  expect(exists).toBe(true)
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)